include/kindr/vectors/Vector.hpp

```diff
@@ -395,6 +437,16 @@ class Vector : public VectorBase<Vector<PhysicalType_, PrimType_, Dimension_> >,
     return *this;
   }
 
+  /*! \brief Multiplication with a scalar and assignment
+   * @param factor  factor with physical type
+   * @returns reference
+   */
+  template <PhysicalType PhysicalTypeFactor_, typename PrimTypeFactor_>
+  Vector<PhysicalType_, PrimType_, Dimension_>& operator*=(Vector<PhysicalTypeFactor_, PrimTypeFactor_, 1> factor) {
```

Comment:
Shouldn't `*=` and `/=` only work if the factor is of physical type `Typeless`? Otherwise we would need to change the type of `this`, right? (As this PR is already imported into our internal mono-repo, just let me know your opinion, and I will do potential adaptations internally.)